### ISSUE TITLE
The module doesn't work

### ISSUE THREAD

**Issue by rockse** [heroku/kafka-helper#3]
It doesn't install dependencies required by kafka-helper which it's not working 

For e.g.

```
File "/app/.heroku/python/lib/python3.6/site-packages/kafka_helper.py", line 9, in <module>
from urlparse import urlparse
ModuleNotFoundError: No module named 'urlparse'
```

**Comment by aschriner** [comment]
It looks like the latest version 70aa539540e7d6517f0c3c1900617767ab198b6c in which @lambacck included a fix for this issue is not the version installed as 0.1 by pip.  I'm not super familiar with what needs to be done to update the package on pypi but I think that's what needs to happen.

in the meantime you can do 
`$ pip install git+git://github.com/heroku/kafka-helper.git@70aa539540e7d6517f0c3c1900617767ab198b6c`

**Comment by alouie-sfdc** [comment]
Thanks -- this should be fixed with the release of kafka-helper 0.2: https://pypi.org/project/kafka-helper/0.2/

**Comment by Swapratim** [comment]
Hi,

I have faced the exact same problem. In this case the culprit is 'flask_oauth' which is not compatible with python 3.7.1.

File "/app/.heroku/python/lib/python3.7/site-packages/flask_oauth.py", line 13, in <module>
from urlparse import urljoin
ModuleNotFoundError: No module named 'urlparse'

I found the fix: pip install git+https://github.com/mitsuhiko/flask-oauth

This works in local absolutely fine but while pushing into heroku-git, again facing the same problem.

Please help.

Thanks in advance!
